weblate/accounts/notifications.py

```diff
@@ -555,7 +555,7 @@ def send_parse_error(profile, component, translation, error, filename):
 def send_mails(mails):
     """Send multiple mails in single connection."""
     try:
-        connection = get_connection()
+        connection = get_connection(fail_silently=True)
```

Comment:
The intention here is to fail verbosely, the SMTP errors are handled below.

Reply:
Hey, this is also just a temporary change for testing purposes.